ui/src/app/admin/layout.tsx

```diff
@@ -7,7 +7,7 @@ export default async function AdminLayout({
   children: React.ReactNode;
 }) {
   const session = await auth();
-  if (!session) return redirect("/api/auth/signout");
+  if (!session) return redirect("/api/signout");
```

Comment:
This should be handled in the middleware instead

Reply:
I agree

Reply:
https://github.com/littlehorse-enterprises/lh-user-tasks/pull/18

I'm waiting for this to be merged as it has a custom `middleware.ts` file and we can update this PR to check if the token is valid